FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.DataClient/Api/FhirApiConstants.cs

```diff
@@ -46,6 +46,6 @@ public static class FhirApiConstants
         /// Currently _count is limited to 1000 in FHIR server.
         /// https://docs.microsoft.com/en-us/azure/healthcare-apis/fhir/overview-of-search#search-result-parameters
         /// </summary>
-        public const int PageCount = 1000;
+        public const int DefaultPageCount = 1000;
```

Comment:
Maybe we can use a new Enum FhirApiPageCount { Single = 1, Batch = 1000 }

Reply:
Good Suggestion!